src/backend/expungeservice/database/database.py

```diff
@@ -0,0 +1,96 @@
+"""Database module"""
+
+import logging
+
+import psycopg2
+import psycopg2.extras
+
+
+class Database(object):
+    """Database connection class.
+
+Example usage:
+
+import os
+password = os.environ['POSTGRES_PASSWORD']
+from expungeservice.database import database
+
+# Get a connection.
+db = database.Database(host='db', port='5432', name='record_expunge',
+                       username='postgres', password=password)
+
+# Run a query.
+query = 'SELECT * FROM users;'
+db.cursor.execute(query, ())
```

Comment:
Why do we pass an empty tuple as second arg?

Reply:
@a-ilango If there are any arguments for the SQL string, they're passed in as a tuple instead of using conventional string-formatting, for security against SQL injection. Strictly speaking the tuple is optional if there are no arguments.

Reply:
Thanks!